### DISCUSSION TITLE
Planning a release anytime soon?

### DISCUSSION THREAD

**Discussion by tmaszk** [tfwright/live_admin#117] [General]
Looks like you've made some nice improvements recently, but haven't had a release in 3 months or so.  I'd like to update, but am hesitant to use a non-released version in production.

**Comment by tfwright** [comment]
Soon, I promise! Just polishing up some of the new features and doing more internal testing. Hoping to release in the next couple of weeks.

**Comment by tfwright** [comment]
Obviously this is taking longer than I thought, there are a few different new APIs in development that I'm still working on. Current code should be pretty stable though. I decided to release a dev version, if you feel better about using that vs a git ref: https://hex.pm/packages/live_admin/0.13.0-dev
